charts/grafana-stakewise-dashboards/values.yaml

```diff
@@ -0,0 +1,9 @@
+## Grafana dashboards folder
+##
+dashboardsFolder: StakeWise
+
+importFromValue:
```

Comment:
let's provide a default value here

Reply:
We don't need default value here, it's for additional dashboards you want to import, all dashboards from default set will imported automatically.